modules/monitoring-targets-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * monitoring/understanding-the-monitoring-stack.adoc
+
+[id="monitoring-targets-for-user-defined-projects_{context}"]
+= Monitoring targets for user-defined projects
+
+When monitoring is enabled for user-defined projects, you can monitor:
+
+* Metrics provided through HTTP service endpoints in user-defined projects.
```

Comment:
why we emphasize `HTTP` service?

Reply:
Thank you for highlighting this. I have now removed 'HTTP' from this line.